test/e2e/framework/sessionconfig.go

```diff
@@ -237,18 +238,29 @@ func (cfg SessionConfig) DeleteFARs() []*ie.IE {
 	}
 }
 
-func (cfg SessionConfig) SessionIEs() []*ie.IE {
+func (cfg SessionConfig) CreateURR (id uint32, triggers uint16) *ie.IE {
+	urr := ie.NewCreateURR(
+		ie.NewURRID(id),
+		ie.NewMeasurementMethod(0, 1, 1), // VOLUM=1 DURAT=1
+		ie.NewReportingTriggers(triggers))
+	// for Quota Validity Time report trigger is 15th bit in host order
+	if (triggers & 0x80) != 0 {
+		urr.Add(ie.NewQuotaValidityTime(time.Now()))
```

Comment:
If this is a dummy time value to be replaced later, you could probably just use `time.Time{}` instead of `time.Now()`